tests/fixtures/joint.py

```diff
@@ -55,6 +58,9 @@
     decryption_secret_id="some-secret",
 )
 
+SignedToken = str
+PubKey = str
+
 
 def get_free_port() -> int:
```

Comment:
By the way, I think this function is not needed, since the AsyncClient uses ASGI to access the application and does not use the network. And instead of httpx.AsyncClient you can also use the AsyncTestClient that Kersten has put into the commons lib in v0.3.0 (it's essentially the same, just presets the base url).

Reply:
Seems to work, changed accordingly